typescript/src/redemption.ts

```diff
@@ -406,3 +409,79 @@ export async function getRedemptionRequest(
 
   return redemptionRequests[0]
 }
+
+/**
+ * Finds the oldest active wallet that has enough BTC to handle a redemption request.
+ * @param amount The amount to be redeemed.
+ * @param bridge The handle to the Bridge on-chain contract.
+ * @param bitcoinClient Bitcoin client used to interact with the network.
+ * @param bitcoinNetwork Bitcoin network.
+ * @returns Promise with the wallet details needed to request a redemption.
+ */
+export async function findWalletForRedemption(
+  amount: BigNumber,
+  bridge: Bridge,
+  bitcoinClient: BitcoinClient,
+  bitcoinNetwork: BitcoinNetwork
+): Promise<{
+  walletPublicKey: string
+  mainUTXO: UnspentTransactionOutput
+}> {
+  const wallets = await bridge.getNewWalletRegisteredEvents()
```

Comment:
Are those events sorted from oldest to newest by default? Can we confirm that? It looks like `getEvents` method from `ethereum-helpers` has `fromBlock` argument which indicates that it starts searching from the given block and goes "up". Does that also mean that the results are returned in the similar order (from oldest to newest)?

Reply:
Yes, I can confirm that they are sorted from oldest to newest by default. But actually, I can't find this in docs 😄.

Reply:
Should I address this here or opened a separate PR?

Reply:
Seems like a simple change that we can include in this PR.